### PR TITLE
Gender from String to ENUM. Breaking Change

### PR DESCRIPTION
**Pull request by SkylerSeamans**

### CODE REVIEW

File: TeamSnapSDK/SDK/DataTypes/TSDKMember.m [hunk 1]
```diff
@@ -119,6 +119,40 @@ -(NSInteger)age {
...
+    NSString *genderString = [self getString:@"gender"];
+    return [self genderFromString:genderString];
+}
+
+- (void)setGender:(TSDKMemberGender)gender {
+    NSString *genderString = [self stringFromGender:gender];
+    [self setString:genderString forKey:@"gender"];
+}
+
+- (TSDKMemberGender)genderFromString:(NSString *)string {
```
Comment (jrahaim) [inline TeamSnapSDK/SDK/DataTypes/TSDKMember.m]: Should this be a Class method?

File: TeamSnapSDK/SDK/DataTypes/TSDKMember.m [hunk 2]
```diff
@@ -119,6 +119,40 @@ -(NSInteger)age {
...
+    if([string.lowercaseString isEqualToString:@"female"]) {
+        return TSDKMemberGenderFemale;
+    } else if([string.lowercaseString isEqualToString:@"male"]) {
+        return TSDKMemberGenderMale;
+    } else {
+        return TSDKMemberGenderUnknown;
+    }
+}
+
+- (NSString *)stringFromGender:(TSDKMemberGender)gender {
```
Comment (jrahaim) [inline TeamSnapSDK/SDK/DataTypes/TSDKMember.m]: Should this be a Class Method?

Comment (SkylerSeamans) [inline TeamSnapSDK/SDK/DataTypes/TSDKMember.m]: Agreed

**Comment by SkylerSeamans** [comment]
Updated @jrahaim

**Pull request merged**